testkit/src/main/scala/pureconfig/BaseSuite.scala

```diff
@@ -7,8 +7,8 @@ import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
 
 class BaseSuite
     extends AnyFlatSpec
-    with ConfigConvertChecks
     with Matchers
-    with ConfigReaderMatchers
     with EitherValues
     with ScalaCheckDrivenPropertyChecks
+    with ConfigConvertChecks
```

Comment:
IntelliJ didn't like these two traits appearing before the first four (which they require), so I just reordered them.